ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -587,6 +615,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-fullyprivate-internal-registry-arm-f14
```

Comment:
This one should be moved to line 588, and replace it?

Reply:
We should move the job behind azure-ipi-fullyprivate-internal-registry-amd-f28-destructive to make them in order.

Reply:
and the pair of `azure-ipi-fullyprivate-internal-registry` jobs should be placed before `azure-ipi-fullyprivate-proxy` jobs.

Reply:
@wewang58 pls do not forget this comments.

Reply:
@jianlinliu  Oh, updated